src/Microsoft.PowerShell.Commands.Utility/commands/utility/AddType.cs

```diff
@@ -111,75 +78,65 @@ public sealed class AddTypeCommand : PSCmdlet
         #region Parameters
 
         /// <summary>
-        /// The source code of this type.
+        /// The source code of this generated type.
         /// </summary>
         [Parameter(Mandatory = true, Position = 0, ParameterSetName = "FromSource")]
         public String TypeDefinition
         {
             get
             {
-                return sourceCode;
+                return _sourceCode;
             }
             set
             {
-                sourceCode = value;
+                _sourceCode = value;
             }
         }
 
         /// <summary>
-        /// The name of the type used for auto-generated types.
+        /// The name of the type (class) used for auto-generated types.
         /// </summary>
         [Parameter(Mandatory = true, Position = 0, ParameterSetName = "FromMember")]
         public String Name { get; set; }
 
         /// <summary>
-        /// The source code of this method / member.
+        /// The source code of this generated method / member.
         /// </summary>
         [Parameter(Mandatory = true, Position = 1, ParameterSetName = "FromMember")]
         [SuppressMessage("Microsoft.Performance", "CA1819:PropertiesShouldNotReturnArrays")]
         public String[] MemberDefinition
         {
             get
             {
-                return new string[] { sourceCode };
+                return new string[] { _sourceCode };
             }
             set
             {
-                sourceCode = "";
+                _sourceCode = String.Empty;
 
                 if (value != null)
                 {
-                    sourceCode = String.Join("\n", value);
+                    _sourceCode = String.Join("\n", value);
                 }
             }
         }
 
-        internal String sourceCode;
+        private String _sourceCode;
 
         /// <summary>
-        /// The namespaced used for the auto-generated type.
+        /// The namespace used for the auto-generated type.
         /// </summary>
         [Parameter(ParameterSetName = "FromMember")]
-        [Alias("NS")]
         [AllowNull]
-        public String Namespace
-        {
-            get
-            {
-                return typeNamespace;
-            }
-            set
-            {
-                typeNamespace = value?.Trim();
-            }
-        }
-
-        internal string typeNamespace = "Microsoft.PowerShell.Commands.AddType.AutoGeneratedTypes";
+        [Alias("NS")]
+        public String Namespace  = "Microsoft.PowerShell.Commands.AddType.AutoGeneratedTypes";
```

Comment:
Please don't change a public property to a field

Reply:
Fixed.